homeassistant/components/recorder/models.py

```diff
@@ -131,13 +131,9 @@ def entity_ids(self, point_in_time=None):
         Specify point_in_time if you want to know which existed at that point
         in time inside the run.
         """
-        from sqlalchemy.orm.session import Session
+        from homeassistant.components import recorder
```

Comment:
Ok, will remove the dependency and just do `scoped_session` in models. I think this is the source of the shutdown warning about threads